snapshot/doc.go

```diff
@@ -2,13 +2,12 @@
 package snapshot offers access to Scoot data.
 Scoot data is largely snapshots (hence the package name).
 
-The main entry point is the interface DB. DB holds Values which can be a Snapshot or a SnapshotWithHistory.
+The main entry point is the interface DB. DB holds Snapshots which can be an FSSnapshot or a GitCommitSnapshot.
 
-A Snapshot is a snapshot of filesystem state (like a tar file).
+An FSSnapshot is a snapshot of filesystem state (like a tar file).
 
-A SnapshotWithHistory is a Snapshot and also a commit message and an optional list of parent SnapshotWithHistory's.
-(This definition corresponds with a git commit, which is convenient for creating git repos.)
+A GitCommitSnapshot mirrors a git commit: a Snapshot, commit metadata, and an optional list of parent GitCommitSnapshots.
```

Comment:
It also includes the history, because various tools that want to use Scoot for data do need the history (this was covered in the SnapshotDB/GitDB design doc)